tests/conftest.py

```diff
@@ -201,9 +200,6 @@ def sync_plan():
         return list(set(sync_ids)), sat_hostname
 
     def teardown_sync_plan():
-        teardown = ansible_module.command(MaintenanceMode.stop())
```

Comment:
@Gauravtalreja1 `setup_sync_plan` is also used by `test_positive_maintenance_mode`. 
Can you see if you need to write separate fixture for it.
This was probably added to makes sure that Satellite will be in consistent state and other tests won't fail in case these two tests failed.

Reply:
you're right here to keep this, there was a failure in `MaintenanceMode.stop()` which got fixed in recent snaps, so closing this PR as its not required now.